packages/megasync-deb/megasync-deb.pacscript

```diff
@@ -0,0 +1,47 @@
+name="megasync-deb"
+gives="megasync"
+repology=("project: megasync")
+version="4.9.4"
```

Comment:
The full version is _4.9.4-3.1_ but using it, _repology_ versions become incompatible. For _21.10_ and _21.04_ the version is different `_4.9.4-3.1_` but adding the version variable inside case statements had no effect!